build.gradle.kts

```diff
@@ -28,6 +28,9 @@ plugins {
 // Gradle docs at https://docs.gradle.org/current/dsl/org.gradle.api.Project.html#org.gradle.api.Project:allprojects(groovy.lang.Closure)
 allprojects {
     repositories {
+        // Uncomment mavenLocal to use local versions of libraries.
```

Comment:
@BottleRocket-Colin Appreciate the comment here! Would you mind adding a quick blurb in the README.md about local launchpad development as well to enhance visibility?